### ISSUE TITLE
onPress

### ISSUE THREAD

**Issue by ksetrin** [PaulBGD/react-native-image-slider#58]
<ImageSlider
loop={false}
loopBothSides={false}
images={this.props.availableImages}
onPress={({ index }) => console.log(index)}
...

onPress doesn't work here. How to handle it?

**Comment by ArtiomShapovalov** [comment]
Hey! Try to use alert(index) or console.warn(index). In my case, onPress works fine. But If you are using custom slides, you should implement onPress inside your custom component.

**Comment by ksetrin** [comment]
Yes, you are right, it's my mistake. I really use custom slides, and that's why and can't handle onPress 
Thank you for responce
